.gitignore

```diff
@@ -1,2 +1,3 @@
 .stack-work/
 *.cabal
+.dir-locals.el
```

Comment:
Pls no, what the hell is this. Add it to your `~/.gitexclude` once and for all.

Reply:
Agree, only project-specific files should be included in project `.gitignore`.